### PR TITLE
Add action to cleanup merged branches

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
Whoops! - thanks for the reminder

File: .github/main.workflow [hunk 1]
```diff
@@ -11,3 +11,14 @@ action "Check CHANGELOG" {
...
 }
+
+
+workflow "on pull request merge, delete the branch" {
+  on = "pull_request"
+  resolves = ["branch cleanup"]
+}
+
+action "branch cleanup" {
+  uses = "jessfraz/branch-cleanup-action@master"
```
Comment (mcfadden8) [inline .github/main.workflow]: Is this user correct?

Comment (davidbeckingsale) [reply .github/main.workflow]: Yes, indeed. See link I sent in Slack.

Comment (mcfadden8) [reply .github/main.workflow]: I suggest we use a fork

**Pull request merged**